packages/mdc-shape/README.md

```diff
@@ -0,0 +1,107 @@
+<!--docs:
+title: "Shape"
+layout: detail
+section: components
+excerpt: "Shapes direct attention, identify components, communicate state, and express brand."
+path: /catalog/shape/
+-->
+
+# Shape
+
+<!--<div class="article__asset">
+  <a class="article__asset-link"
+     href="https://material-components-web.appspot.com/shape.html">
+    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/shape.png" width="<TODO>" alt="Shape screenshot">
```

Comment:
Does the <TODO> need to be filled in?

Reply:
Yup. I didn't have a value before but I have one now.